Makefile

```diff
@@ -8,15 +8,26 @@ IMAGE_TAG ?= latest
 PULL_POLICY ?= Always
 STATE_HANDLER_IMAGE ?= kubernetes-nmstate-state-handler
 POLICY_HANDLER_IMAGE ?= kubernetes-nmstate-configuration-policy-handler
+KUBECONFIG ?= $(shell pwd)/cluster/.kubeconfig
+FUNC_TESTS = tests/nmstate_suite_test.go tests/reporting_state_test.go
+FUNC_TEST_ARGS ?= ""
 
 commands = state-handler policy-handler
+func_tests_sources = $(shell find tests/ -name "*.go")
 
 $(commands):
 	go fmt ./cmd/$@
 	go vet ./cmd/$@
 	go build -o ./bin/$@ ./cmd/$@ 
 
-build: policy-handler state-handler
+build: $(commands)
+
+build:
```

Comment:
how does this behave? when you have two targets with the same name

Reply:
It runs both, let remove the old one